src/lib_generated.rs

```diff
@@ -6,7 +6,7 @@ use core::mem::MaybeUninit;
 
 pub use crate::error::Error;
 pub type Result<T, E = Error> = core::result::Result<T, E>;
-pub type Size = usize;
+pub type Size = u32;
```

Comment:
Was this intentional? IIRC this was a convenience from before because `usize` is used so ubiquitously in Rust right nwo